app/assets/javascripts/govuk_publishing_components/components/details.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-env jquery */
+// = require govuk/components/details/details.js
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  'use strict'
+
+  Modules.GovukDetails = function () {
+    this.start = function (element) {
+      var customTrackLabel = element[0].getAttribute('data-track-label')
+
+      // If a custom label has been provided, we can simply call the tracking module
+      if (customTrackLabel) {
+        var trackDetails = new window.GOVUK.Modules.TrackClick()
+        trackDetails.start(element)
+      } else {
+        // If no custom label is set, we use the open/close status as the label
+        var detailsComponent = element[0]
+        var detailsClick = detailsComponent.querySelector('[data-details-track-click]')
+        var that = this
+
+        $(detailsClick).click(function (e) {
```

Comment:
Is this line jquery? Only it looks like the rest of the file doesn't use it, so would be good to avoid it entirely if possible.

Reply:
This was originally written without jQuery, but we seem to have a mismatch in the way we write tests (relying heavily on jQuery). I had an issue with tests passing locally, but failing in CI. It seemed to be because of this mismatch, so the click event wasn't being triggered. The simplest fix was to move this line to jQuery - not ideal, but moving everything the other way (tests written without jQuery) looked like a bigger piece work. 

I've made a note, so will revisit it once this has been deployed.

Reply:
Fair enough.